pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -572,3 +598,51 @@ func getChaosType() error {
 	}
 	return nil
 }
+
+// reconcileForDelete
+func (r *ReconcileChaosEngine) reconcileForDelete(request reconcile.Request) (reconcile.Result, error) {
+
+	r.recorder.Eventf(engine.Instance, corev1.EventTypeNormal, "Stopping reconcile for chaosEngine", "Will remove all Chaos resources for chaosEngineUUID: %v", string(engine.Instance.UID))
+	optsDelete := []client.DeleteAllOfOption{
+		client.InNamespace(request.NamespacedName.Namespace),
+		client.MatchingLabels{"engineUUID": string(engine.Instance.UID)},
+	}
+	reqLogger := chaosTypes.Log.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	err := r.client.DeleteAllOf(context.TODO(), &corev1.Pod{}, optsDelete...)
+	if err != nil {
+		reqLogger.Error(err, "Unable to delete chaosEngine allocated Chaos Resources")
+		return reconcile.Result{}, err
+	}
+	opts := client.UpdateOptions{}
+	engine.Instance.Spec.EngineStatus = "stopped"
```

Comment:
what's the use of this can you explain?

Reply:
Its just that to show that reconciler/operator is changing the values. I'm changing started --> initialized, and stop --> stopped